ts/state/selectors/messages.ts

```diff
@@ -174,3 +183,110 @@ export function useHideAvatarInMsgList(messageId?: string, isDetailView?: boolea
 export function useMessageSelected(messageId?: string) {
   return useSelector((state: StateType) => getIsMessageSelected(state, messageId));
 }
+
+/**
+ *  ==================================================
+ *  Below are selectors for community invitation props
+ *  ==================================================
+ */
+
+/**
+ * Return the full url needed to join a community through a community invitation message
+ */
+export function useMessageCommunityInvitationFullUrl(messageId: string) {
+  return useMessagePropsByMessageId(messageId)?.propsForGroupInvitation?.fullUrl;
+}
+
+/**
+ * Return the community display name to have a guess of what a community is about
+ */
+export function useMessageCommunityInvitationCommunityName(messageId: string) {
+  return useMessagePropsByMessageId(messageId)?.propsForGroupInvitation?.serverName;
+}
+
+/**
+ *  =========================================
+ *  Below are selectors for call notification
```

Comment:
Can you correct these comments to either 

1. Below are selectors for call notifications
2. Below are selectors for the call notification

As you want

Reply:
fixed